MatrixKit/Models/Room/MXKAttachment.m

```diff
@@ -80,34 +81,42 @@ - (instancetype)initWithEvent:(MXEvent *)mxEvent andMatrixSession:(MXSession*)mx
         // Set default thumbnail orientation
         _thumbnailOrientation = UIImageOrientationUp;
         
-        NSString *msgtype =  eventContent[@"msgtype"];
-        if ([msgtype isEqualToString:kMXMessageTypeImage])
+        if (mxEvent.eventType == MXEventTypeSticker)
         {
-            _type = MXKAttachmentTypeImage;
-        }
-        else if ([msgtype isEqualToString:kMXMessageTypeAudio])
-        {
-            _type = MXKAttachmentTypeAudio;
-        }
-        else if ([msgtype isEqualToString:kMXMessageTypeVideo])
-        {
-            _type = MXKAttachmentTypeVideo;
-            
+            _type = MXKAttachmentTypeSticker;
             _thumbnailInfo = eventContent[@"info"][@"thumbnail_info"];
```

Comment:
We should use`MXJSONModelSetXxx` helpers to avoid crash because of malformed content.
But that comment is for all this method.